models/inputs.yaml

```diff
@@ -20,15 +20,12 @@ inputs:
         - select: "user_id"
           type: user_id
           entity: user
-          to_default_stitcher: true
```

Comment:
.DS_Store file should not be part of this PR

Reply:
Removed.

Reply:
I still see it in the PR.

Reply:
@shekhar-rudder you can see that the file is now in bin: https://github.com/rudderlabs/rudderstack-profiles-base-features/pull/29/commits/f6690abff00c84e6b766193890ba7e26469cb813

Reply:
My bad. I didn't notice that it is an already committed file